### ISSUE TITLE
Add Edit button to Search Results listings if permissions allow

### ISSUE THREAD

**Issue by wickr** [OregonDigital/oregondigital_2#154]
It'd be nice to have an Edit button on listings in Search Results, it would save a step when opening the item and then clicking Edit.

Of course this should only show up if permissions allow it.

**Comment by terrellt** [comment]
Once #155 is merged this should be a 3-line addition.
